proxy/ReverseProxy.cc

```diff
@@ -67,7 +67,7 @@ init_reverse_proxy()
 
   Note("%s loading ...", ts::filename::REMAP);
   if (!rewrite_table->load()) {
-    Fatal("%s failed to load", ts::filename::REMAP);
+    Warning("%s failed to load", ts::filename::REMAP);
```

Comment:
should we return early here? So we don't see a "finished loading" in the failure case?

Reply:
+1 makes sense.